Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageViewModel+ChangeType.swift

```diff
@@ -19,14 +19,16 @@ extension EditorPageViewModel {
             case .string:
                 fallthrough
             case .otherContent:
-                if let type = try? objectTypeProvider.defaultObjectType(spaceId: document.spaceId) {
-                    onChangeType(type: type, pasteContent: true)
+                Task {
```

Comment:
If it possible, in future, remote to delete nested Task in all places. If you make `onChangeType(typeSelection:` as `async`, you also make `async` internal method `onChangeType(type: ObjectType)`
This is better for prevent the bugs